src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,93 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int RESIZE_INDEX = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size;
+    private int threshHold;
+    private Node<K, V>[] hashTable;
+
+    public MyHashMap() {
+        hashTable = new Node[DEFAULT_CAPACITY];
+        threshHold = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        resize();
+        int index = getIndex(key);
+        if (hashTable[index] == null) {
+            Node<K, V> newNode = new Node<>(key, value);
+            hashTable[index] = newNode;
+            size++;
+        }
+        Node<K, V> node = hashTable[index];
+        while (node != null) {
+            if (Objects.equals(node.key, key)) {
+                node.value = value;
+                return;
+            }
+            node = node.next;
+        }
+        Node<K, V> newNode = new Node<>(key, value);
+        newNode.next = hashTable[index];
+        hashTable[index] = newNode;
+        size++;
     }
 
     @Override
     public V getValue(K key) {
+        Node<K, V> getNode = hashTable[getIndex(key)];
+        while (getNode != null) {
+            if (Objects.equals(getNode.key, key)) {
+                return getNode.value;
+            }
+            getNode = getNode.next;
+        }
```

Comment:
Optional. you have similar block of code. I'd recommend you creating new method and put similar logic there.